Lesson5 (Ordered associative containers)/TestFunctions.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <vector>
+#include <set>
+#include <sstream>
+#include <algorithm>
+#include <map>
+#include "cctype"
+
+
+
+std::vector<std::string> getUniqueWords (const std::string& text)
+{
+    std::vector<std::string> outpVector;
+    std::string copyText = text;
+    std::set<std::string> setOfWords;
+    std::replace_if(copyText.begin(), copyText.end(), [](unsigned char c){ return std::ispunct(c); }, ' ');
+    std::stringstream streamText(copyText);
+    while (streamText >> copyText)
+    {
+        std::transform(copyText.begin(), copyText.end(), copyText.begin(), tolower);
+        auto it = setOfWords.insert(copyText);
+        if (it.second == true)
+        {
+            outpVector.emplace_back(copyText);
```

Comment:
i think there is a better solution to move the whole set to vector it via move_iterator and assigning constructor